executor_manager/services/sandbox/execution_runner.py

```
@@ -0,0 +1,241 @@
+        except asyncio.TimeoutError:
+            logger.warning(
+                f"[ExecutionRunner] Execution request timeout: "
+                f"execution_id={execution.execution_id}"
+            )
+            return False, "Executor container not responding (timeout)"
+
+        except httpx.ConnectError as e:
+            logger.error(
+                f"[ExecutionRunner] Cannot connect to executor: "
+                f"execution_id={execution.execution_id}, error={e}"
+            )
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)